package.json

```diff
@@ -2,7 +2,7 @@
   "name": "language-haskell",
   "displayName": "Haskell Syntax Highlighting",
   "description": "Syntax support for the Haskell programming language.",
-  "version": "3.3.0",
+  "version": "3.5.0",
```

Comment:
Wasn't sure why this was still at 3.3 when the version available in VScode is 3.4, but figured I would change it. Let me know if this isn't right